main.py

```diff
@@ -40,4 +41,16 @@ def run(self):
             else:
                 cmd.append('{}'.format(path))
 
-        subprocess.Popen(cmd)
+        startupinfo = None
+        if os.name == "nt":
+            startupinfo = subprocess.STARTUPINFO()
+            startupinfo.dwFlags |= subprocess.STARTF_USESHOWWINDOW
+
+        subprocess.Popen(
+            args=cmd,
```

Comment:
I tested this om my mac and It did not open the file or add the folder to the project. I replaced this line with `args=" ".join(cmd)`, and that works. I remember reading something about that it's better to pass a array instead of a string but I can't recall.